modules/compliance-tailored-profiles.adoc

```diff
@@ -74,6 +77,24 @@ spec:
 |`setValues`
 | A list of name, rationale, and value groupings. Each name refers to a name of the value set. The rationale is human-readable text describing the set. The value is the actual setting.
 |===
++
+.. Add the `tailoredProfile.spec.manualRules` attribute :
++
+.Example `tailoredProfile.spec.manualRules.yaml`
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: TailoredProfile
+metadata:
+  name: rhcos4-manual-scc-check
+spec:
+  extends: ocp4-cis
+  description: This profile extends ocp4-cis by forcing the SCC check to always return MANUAL
+  title: OCP4 CIS profile with manual SCC check
+  manualRules:
+    - name: ocp4-cis-scc-limit-container-allowed-capabilities
+    rationale: We use third party software that installs its own SCC with extra privileges
```

Comment:
Oops, there were two more issues in the example I provided, sorry about that :-(
the name should be `ocp4-scc-limit-container-allowed-capabilities` and the `rationale` attribute should be indented so that it starts on the same column as the `name` attribute This is the full example that I even tested this time:
```
apiVersion: compliance.openshift.io/v1alpha1
kind: TailoredProfile
metadata:
  name: ocp4-manual-scc-check
spec:
  extends: ocp4-cis
  description: This profile extends ocp4-cis by forcing the SCC check to always return MANUAL
  title: OCP4 CIS profile with manual SCC check
  manualRules:
    - name: ocp4-scc-limit-container-allowed-capabilities
      rationale: We use third party software that installs its own SCC with extra privileges
```
sorry I made you redo the PR once again, but the text looks good to me now!

Reply:
@jhrozek It is no problem at all! Updates have been made.